domain/crossmodelrelation/service/remoteapplication.go

```diff
@@ -252,8 +203,22 @@ func (s *Service) GetRemoteApplicationConsumers(ctx context.Context) ([]crossmod
 
 // SetRemoteApplicationOffererStatus sets the status of the specified remote
 // application in the local model.
```

Comment:
Won't this return an application not found error if the uuid does not exist?
We need to document it.